graphql/introspection/testdata/TestComputeSchemaJSON.snapshots.json

```diff
@@ -128,6 +130,45 @@
               "name": "Asset",
               "possibleTypes": []
             },
+            {
+              "description": "",
+              "enumValues": [],
+              "fields": [
+                {
+                  "args": [],
+                  "deprecationReason": "",
+                  "description": "",
+                  "isDeprecated": false,
+                  "name": "enum",
+                  "type": {
+                    "kind": "NON_NULL",
+                    "name": null,
+                    "ofType": {
+                      "kind": "ENUM",
+                      "name": "enumType",
+                      "ofType": null
+                    }
+                  }
+                },
+                {
+                  "args": [],
+                  "deprecationReason": "",
+                  "description": "",
+                  "isDeprecated": false,
+                  "name": "maybeEnum",
+                  "type": {
+                    "kind": "SCALAR",
+                    "name": "int32",
+                    "ofType": null
+                  }
```

Comment:
I think this should be some kind of nullable ENUM, instead it becomes a scalar of type int32